tests/test_wrapper.py

```diff
@@ -319,28 +319,29 @@ def test_file_close_on_exception():
 
 @pytest.mark.skipif(sys.version_info >= (3, 0, 0),
                     reason='Previous release requires Python2')
-def test_conversion_script():
+def test_conversion_script(tmpdir):
     try:
-        import h5py_wrapper_001.wrapper as h5w_001
+        import h5py_wrapper_101.wrapper as h5w_101
```

Comment:
why did the version change from 001 to 101 in this PR?

Reply:
Originally, we wrote this test after releasing 0.0.1. After releasing 1.0.1., we did not change this here, although I think that we should have, because in my understanding, this test should test against the previous release. Or against all previous releases?

Reply:
I think we should provide conversion from alll old versions to the current version, so this test should check for all previous releases. Is this possible with small changes? Otherwise we should handle this in a new PR.